leptonai/cli/workspace.py

```diff
@@ -132,6 +168,20 @@ def remove(workspace_id: str):
     console.print(f"Successfully removed workspace: [green]{workspace_id}.[/]")
 
 
+@workspace.command()
+# @click.option("--workspace-id", "-i", help="ID of the workspace", required=True)
```

Comment:
This seems to be a half-finished function? It should not take any options and should simply remove all workspaces.

Reply:
Yes, this PR is a work in progress. I'm waiting for the 404 case you mentioned to complete testing. The removeAll function is currently used for my test utility, as the end-to-end tests generate excessive dummy workspace information. I need this function for testing and will remove it later.